packages/quick_actions/quick_actions/README.md

```diff
@@ -34,7 +34,7 @@ Finally, manage the app's quick actions, for instance:
 ```dart
 quickActions.setShortcutItems(<ShortcutItem>[
   const ShortcutItem(type: 'action_main', localizedTitle: 'Main view', icon: 'icon_main'),
-  const ShortcutItem(type: 'action_help', localizedTitle: 'Help', icon: 'icon_help')
+  const ShortcutItem(type: 'action_help', localizedTitle: 'Help', localizedSubtitle: 'Tap to get help', icon: 'icon_help')
```

Comment:
It's not clear to me that this is a common enough option that we actually need to update the README and example app to demonstrate it.